package.json

```diff
@@ -39,7 +39,7 @@
     "postinstall": "yarn migrate:up",
     "build": "webpack -p",
     "start": "node index.js",
-    "debug": "node debug index.js",
+    "debug": "node --inspect index.js",
```

Comment:
works much more nicely -- it gives you a url to open in Chrome so you can use the Chrome debugger instead of node's CLI debugger